frontend/cypress/integration/multiPart.spec.js

```diff
@@ -0,0 +1,296 @@
+import faker from 'faker'
+const syncedRepoUrlMultiParts =
+  'https://github.com/kitspace-forks/DIY_particle_detector'
+const syncedRepoUrl = 'https://github.com/kitspace-forks/CH330_Hardware'
+const multiPartsNames = ['alpha-spectrometer', 'electron-detector']
+const multiPartsRepoName = syncedRepoUrlMultiParts.split('/').slice(-1).toString()
+const normalRepoName = 'CH330_Hardware'
+
+describe('Render project cards', () => {
+  before(() => {
+    // visit home before running the tests, instead of using `wait`.
```

Comment:
I don't understand this comment. Why would you need wait, how does a `visit` replace a a `wait`? Might be better without any comment.

Reply:
The purpose of this isn't actually visiting the homepage. Sometimes, the frontend has a slow startup time which results in a failure for the first few tests.

Reply:
Ok, can you write that in the comment please.

Reply:
Resolved in 86336a6.